src/tests/particles/products-test.ts

```diff
@@ -69,7 +83,7 @@ describe('products test', () => {
             const verified = content.model && content.model.hasItems
                 && content.model.items['$template'].length > 0
                 && 1 === content.model.items.models.length;
-            if (verified) {
+            if (verified && !Flags.useNewStorageStack) {
```

Comment:
I think we need to verify that the value is right when useNewStorageStack is set too.

Reply:
removed the check as discussed as I think it is redundant with the check in verifyFilteredBook. Maria can you check if this is ok?

Reply:
the check you removed used to verify the rendered content, but that would become redundant anyway with the migration to uibroker, so it's ok to remove it. thanks for checking.